### PR TITLE
Update Chinese(zh_CN)

### PR DESCRIPTION
**Pull request by xiaopangju**

### CODE REVIEW

File: zh_CN/messages.json [hunk 1]
```diff
@@ -846,7 +852,7 @@
         "message": "上一行"
     },
     "Line_break": {
```
Comment (xiaopangju) [inline zh_CN/messages.json:854]: @derjanb Can you give me the context of this string? This way I can check whether my own translation is correct.

Comment (derjanb) [reply zh_CN/messages.json:854]: It is an editor option as to whether long lines should be automatically wrapped.

Comment (xiaopangju) [reply zh_CN/messages.json:854]: thanks

**Comment by xiaopangju** [comment]
@derjanb You can merge this now! Thanks for your answer

**Comment by derjanb** [comment]
Thank you!

**Pull request merged**